localstack/services/sqs/provider.py

```diff
@@ -1177,10 +1198,19 @@ def send_message_batch(
 
         successful = []
         failed = []
+        batch_message_size = 0
+        max_message_size = queue.maximum_message_size
 
         with queue.mutex:
             for entry in entries:
                 try:
+                    # must convert to unicode to get actual bytes
+                    batch_message_size += len(entry.get("MessageBody").encode("utf8"))
+                    if batch_message_size > max_message_size:
+                        error = "InvalidParameterValue. One or more parameters are invalid. "
+                        error += "Reason: Message must be shorter than 262144 bytes."
+                        raise InvalidParameterValue(error)
```

Comment:
The error related to sending oversized messages in batch seems to behave completely differently. First, it seems to send a `BatchRequestTooLong` error instead of the `InvalidParameterValue` that `send_message` sends. Second, it seems to ignore the queue's maximum message size, and just use the default maximum size